idunn/api/places_list.py

```diff
@@ -58,16 +74,39 @@ def max_size(cls, v):
         sizes = [v, max_size]
         return min(int(i) for i in sizes if i is not None)
 
-    @validator('raw_filter', always=True)
+    @validator('raw_filter')
     def valid_raw_filter(cls, v):
         if "," not in v:
             raise ValueError(f"raw_filter \'{v}\' is invalid")
         return v
 
-def get_places_bbox(bbox, es: Elasticsearch, indices: IndexNames, settings: Settings, query_params: http.QueryParams):
+    @validator('category')
+    def valid_category(cls, v):
+        if v not in ALL_CATEGORIES:
+            raise ValueError(f"Category \'{v}\' is invalid since it does not belong to the set of possible categories: {list(ALL_CATEGORIES.keys())}")
+        else:
+            v = ALL_CATEGORIES.get(v)
+        return v
+
+    @validator('category', whole=True)
+    def flat_categories(cls, v):
+        return [filt for filters in v for filt in filters]
```

Comment:
Since a single category is associated to *a list of raw_filters* (c.f. categories.yml) here the `category` is a list of validated lists : this is why we flat it